notebooks/community/ml_ops/stage7/get_started_with_model_monitoring_xgboost.ipynb

```diff
@@ -0,0 +1,1477 @@
+{
```

Comment:
<p>I would add a new line btw parameters for better readability :)</p><p> </p><pre class="ql-syntax" spellcheck="false">lerting_config = model_monitoring.EmailAlertConfig(
&nbsp;&nbsp;user_emails=[USER_EMAIL], 
  enable_logging=True
  )
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1211/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='49'/>

Reply:
<p>done</p>